proto/storage/version.proto

```diff
@@ -4,14 +4,18 @@ option go_package = "storage";
 option java_package = "io.stackrox.proto.storage";
 
 import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
 
 package storage;
 
 message Version {
     // This is a strictly incrementing integer describing the DB version.
-    int32 seq_num = 1;
+    int32 seq_num = 1 [(gogoproto.moretags) = 'search:"Current Sequence Number"'];
```

Comment:
@c-du Only because I wanted those columns in the database to make debugging simpler and adding a search parameter is how we get things into columns.  This is not something we have to do, but certainly something I prefer to do.

Reply:
@dashrews78 The version store is not gen'd anymore right so we could just avoid this?